scripts/auto-rebase/rebase.sh

```diff
@@ -672,8 +673,22 @@ rebase_to() {
     title "# Rebasing to ${release_image_amd64} and ${release_image_arm64}"
     download_release "${release_image_amd64}" "${release_image_arm64}"
 
-    rebase_branch=rebase-${release_image_amd64#*:}
-    rebase_branch=${rebase_branch%-x86_64}
+    if [[ "${release_image_amd64#*:}" =~ ${RELEASE_TAG_RX} ]]; then
+        ver_stream=${BASH_REMATCH[1]}
+        amd64_date=${BASH_REMATCH[2]}
+    else
+        echo "Failed to match regex against amd64 image tag"
+        exit 1
+    fi
+
+    if [[ "${release_image_arm64#*:}" =~ ${RELEASE_TAG_RX} ]]; then
+        arm64_date="${BASH_REMATCH[2]}"
+    else
+        echo "Failed to match regex against arm64 image tag"
+        exit 1
+    fi
+
+    rebase_branch="rebase-${ver_stream}+amd64-${amd64_date}+arm64-${arm64_date}"
```

Comment:
Is this necessary for creating a branch name? I mean, why not to simply have rebase-${branch}-${timestamp}?

Since the name is transient, it may simplify the code, I think

Reply:
We have this flow in create_pr.py that if no new nightly was produced, but we merged something to microshift's main it'll update already existing branch (and PR) with "rebased" (on newer main) changes

Reply:
Understood. If that's what we need, the code seems to be fine.